man/systemd-tmpfiles.xml

```diff
@@ -149,13 +149,24 @@
         the specified prefix. This option can be specified multiple
         times.</para></listitem>
       </varlistentry>
+
       <varlistentry>
         <term><option>--exclude-prefix=<replaceable>path</replaceable></option></term>
         <listitem><para>Ignore rules with paths that start with the
         specified prefix. This option can be specified multiple
         times.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>-E</option></term>
+        <listitem><para>A shortcut for <literal>--exclude-prefix=/dev --exclude-prefix=/proc
+        --exclude-prefix=/run --exclude-prefix=/sys</literal>, i.e. exclude the hierarchies typically backed
+        by virtual or memory file systems. This is useful in combination with <option>--root=</option>, if
+        the specified directory tree contains an OS tree without these virtual/memory file systems mounted
+        in, as it is typically not desirable to create any files and directories below these subdirectories
+        if they are supposed to be overmounted during runtime.</para></listitem>
```

Comment:
So I was afraid of API break. i.e. this stuff existed for a longer time, and did apply thing to /sys, /proc, and so on... 

I have the suspicion changing this would probably break things, at the very least some tests, where people test that the tmpfiles in some /run tree are properly created...

I mean, you are right, if we'd add --root= today I think we should definitely invert the behaviour, but I have the suspicion it's too late for that.

Or do you feel strong about this? I don't care overly much, I primarily wanted to avoid needless breakage here..

Reply:
You're right, it's has been this way since 2014. I *think* we might still want to change this, but some more research should be done whether it'd affect existing users. So let's use the current version for now.